### DISCUSSION TITLE
Filenames in stacktraces

### DISCUSSION THREAD

**Discussion by kuhnroyal** [getsentry/sentry-dart#697] [Q&A] [answered]
Reading here: https://develop.sentry.dev/sdk/event-payloads/stacktrace/ the filename in a `SentryStackTraceFrame` should be relative to the project root for inApp frames. Currently it seems that it is only the last path segment (filename).

I recently linked my repositories to my Sentry instance but files in stacktraces are not linked. I wonder if that is actually supported. And if it would work if the filenames were correct.

Should this be changed from `app.dart` -> `lib/app.dart`?

**Comment by ueman** [comment, marked as answer]
This is unfortunately already a known issue: https://github.com/getsentry/sentry-dart/issues/347
